### PR TITLE
revising lint code.

### PR DESCRIPTION
**Pull request by jakejh**

### CODE REVIEW

File: workflows/custom-lint.R [hunk 1]
```diff
@@ -30,88 +26,92 @@ doubleQuotesLinter = function(sourceFile) {
...
   if (is.null(repository)) {
-    dirIndexes = gregexpr('/', getwd())[[1]]
-    repository = substr(getwd(), dirIndexes[length(dirIndexes)] + 1, nchar(getwd()))}
+    dirIndexes = gregexpr('/', getwd())[[1L]]
+    repository = substr(getwd(), dirIndexes[length(dirIndexes)] + 1L, nchar(getwd()))}
 
   if (is.null(branch)) branch = 'main'
 
-  lfDt = unique(as.data.table(lintsFound), by = c('filename', 'line_number', 'message'))
+  lfDt = unique(as.data.table(lintsFound), by = c('filepath', 'line_number', 'message'))
```
Comment (JSchoenbachler) [inline workflows/custom-lint.R]: @jakejh I don't see anywhere you are changing this, but `filename` is the name that `lintr` generates. Do you want us to modify the name or just keep as is? I was keeping it this way just so I wasn't actually modifying anything `lintr` gave us back.

**Comment by JSchoenbachler** [comment]
You should run this in R to test it:

```
repository = 'tipa'
branch = 'main'
source('~/{PATH_TO_ACTIONS_REPO}/workflows/custom-lint.R')
lintsFound
lfDt = getLintDt(lintsFound)
issueStr = getFormattedIssueStr(lfDt)
```

Due to the comment I posted above, it errors when I run `getLintDt(lintsFound)`.

**Comment by jakejh** [comment]
Ah, ok, then we can leave it at `filename`.

**Review by JSchoenbachler — APPROVED** [review]
Looks good now!

**Comment by jakejh** [comment]
Make sure to change the file names in the R package repos to "lint_ignore.csv".

**Pull request merged**